src/tutorial/runtime/runtime-configuration.rst

```diff
@@ -59,16 +59,15 @@ Job Submission
 
    By default Cylc runs :term:`jobs <job>` on the machine where the suite is
    running. We can tell Cylc to run jobs on other machines by setting the
-   ``[remote]host`` setting to the name of the host, e.g. to run a task on the
-   host ``computehost`` you might write:
+   ``platform`` setting: If, for example you want to run a task job on a
+   platform called ``powerful_computer`` you would write:
 
 .. code-block:: cylc
 
    [runtime]
        [[hello_computehost]]
            script = echo "Hello Compute Host"
-           [[[remote]]]
-               host = computehost
+           platform = powerful_computer
 
```

Comment:
Shouldn't there be some mention of how a platform is configured? E.g. showing a small global.cylc snippet? Or a link to platforms section elsewhere in the docs (Unless such a section doesn't exist yet)?

Reply:
I'm not sure how much info we want to give. Ideally users should never have to configure a platform. A small number of power users may need to know, but I think most users should be in blissful ignorance.

Reply:
Makes sense. Maybe just use `` :term:`platform` `` so it links to the glossary?